cmd/mb-main.go

```diff
@@ -156,7 +156,7 @@ func mainMakeBucket(cliCtx *cli.Context) error {
 			continue
 		}
 
-		if cliCtx.Bool("s") {
+		if cliCtx.Bool("with-versioning") {
```

Comment:
I was wondering the same when I reviewed the PR that introduced this, but then I saw L132 and L131 so..

Reply:
@vadmeste L132 and L131 are defined shorthands, see L37 & L41. 

The PR deleted `-s` as a valid option, so this no longer works.

Reply:
oh I understand now, thanks!